cypress/e2e/certificate-provider.cy.js

```diff
@@ -0,0 +1,124 @@
+describe('Certificate provider task', () => {
```

Comment:
it felt like now all of this flow is in that a single test for the whole thing would be better than one per page